libs/application/templates/financial-aid/src/lib/FinancialAidTemplate.ts

```diff
@@ -137,14 +137,26 @@ const FinancialAidTemplate: ApplicationTemplate<
             },
           ],
         },
+        on: {
+          SUBMIT: { target: ApplicationStates.SUBMITTED },
+        },
       },
       [ApplicationStates.SUBMITTED]: {
         meta: {
           name: application.name.defaultMessage,
           lifecycle: oneMonthLifeCycle,
           roles: [
             {
-              id: Roles.APPLICANT || Roles.SPOUSE,
+              id: Roles.APPLICANT,
+              formLoader: () =>
+                import('../forms/Submitted').then((module) =>
+                  Promise.resolve(module.Submitted),
+                ),
+              // TODO: Limit this
+              read: 'all',
+            },
+            {
+              id: Roles.SPOUSE,
```

Comment:
was this necessary, did it not work like Roles.APPLICANT || Roles.SPOUSE? Because we are redirecting to the same page?

Reply:
Yeah this was necessary, kinda annoying. It didn´t know where to redirect the spouse otherwise. Also saw other projects doing it like this (children-residence-change):
<img width="509" alt="image" src="https://user-images.githubusercontent.com/43727721/159899145-7837d5b7-18b7-4f5f-a83d-d577a9095382.png">